fabric/src/main/java/com/ultreon/mods/tuem/fabric/ConfigImpl.java

```diff
@@ -7,6 +7,7 @@
 
 public class ConfigImpl {
     public static void register(Object context) {
-        ForgeConfigRegistry.INSTANCE.register(UltimateEnderMod.MOD_ID, ModConfig.Type.CLIENT, Config.build());
+        if(context != null)
```

Comment:
This has to be checked because the server doesn't need to know about client configs

Reply:
Now that I think about it, this is just forge config which doesn't need to exist on fabric

Reply:
> Now that I think about it, this is just forge config which doesn't need to exist on fabric

It needs to exist, otherwise fabric isn't goin to have a config.

Reply:
Also, the if check makes the config never being registered.
Which isn't what is supposed to be done.

Reply:
This is forge specific config (ForgeConfigRegistry) plus the config is never supposed to be registered server side if it's client config (ModConfig.Type.CLIENT)

Reply:
No, this is for Fabric, this is using [Forge Config Api Port Fabric](https://modrinth.com/mod/forge-config-api-port).

Reply:
Also, `ModConfig.Type.CLIENT` can be used in common lol.

Reply:
> Also, `ModConfig.Type.CLIENT` can be used in common lol.

I don't think the server is supposed to be able to know about client side configs?

Reply:
> > Also, `ModConfig.Type.CLIENT` can be used in common lol.
> 
> I don't think the server is supposed to be able to know about client side configs?

Testing this, it creates a client side config on the server which I think is not good

Reply:
The `if(context != null)` check will always be false, since `Config.register` is always called with null on Fabric.

Reply:
I am aware of that, give me a second